allow fine-tuning of timing for lorawan-device

lthiery opened the pull request.

plaes commented:
Could you try building with the `#![allow(missing_docs)]` removed on the first line?

lthiery commented:
> Could you try building with the #![allow(missing_docs)] removed on the first line?

I added that when I moved the module from lorawan-device to lora-phy. There are some undocumented structs...

lucasgranberg commented on lora-phy/src/lorawan_radio.rs:
```diff
@@ -29,22 +34,36 @@ where
     DLY: DelayNs,
 {
-    // TODO: Use a struct with Default or Option fallback?
     fn get_rx_window_offset_ms(&self) -> i32 {
```
Could you add a comment on what this offset and duration actually does?

lthiery replied:
The comment exists on the trait definition already: https://github.com/lora-rs/lora-rs/blob/main/lorawan-device/src/lib.rs#L61-L70

lucasgranberg replied:
Whoops, didn't see that
